solution/shortest_path/18243/main.py

```diff
@@ -0,0 +1,37 @@
+# Authored by : seonjoo0885
+# Co-authored by : -
+# Link : http://boj.kr/4bdf8c52639d4bc1b1074d270f895515
+
+import sys, heapq
+def input():
+    return sys.stdin.readline().rstrip()
+INF = float('inf')
+
+def dijkstra(start):
+    q = [(0, start)]
+    distance = [INF] * n
+    while q:
+        dist, now = heapq.heappop(q)
+        if dist > distance[now]:
+            continue
+        for i in graph[now]:
+            cost = dist + i[1]
+            if distance[i[0]] > cost:
+                distance[i[0]] = cost
+                heapq.heappush(q, (cost, i[0]))
```

Comment:
i[0], i[1]과 같은 변수를 사용하는 것 보다 좀더 명시적인 변수명을 사용합시다.
```suggestion
        for nxt, val in graph[now]:
            cost = dist + val
            if distance[nxt] > cost:
                distance[nxt] = cost
                heapq.heappush(q, (cost, nxt))
```